modules/merge_waiter/main.tf

```diff
@@ -37,23 +46,7 @@ resource "aws_iam_role" "merge_waiter" {
 POLICY
 }
 
-resource "aws_iam_role_policy_attachment" "role-lambda-execution" {
-    role       = "${aws_iam_role.merge_waiter.name}"
-    policy_arn = "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"
-}
-
-resource "aws_iam_role_policy_attachment" "role-lambda-ssm" {
-    role       = "${aws_iam_role.merge_waiter.name}"
-    policy_arn = "arn:aws:iam::aws:policy/AmazonSSMFullAccess"
-}
-
-resource "aws_iam_role_policy_attachment" "role-cloudwatch" {
-    role       = "${aws_iam_role.merge_waiter.name}"
-    policy_arn = "arn:aws:iam::aws:policy/CloudWatchFullAccess"
-}
-
-resource "aws_iam_role_policy_attachment" "role-codedeploy" {
+resource "aws_iam_role_policy_attachment" "role-pipeline-execution" {
     role       = "${aws_iam_role.merge_waiter.name}"
-    policy_arn = "arn:aws:iam::aws:policy/AWSCodeDeployFullAccess"
+    policy_arn = "arn:aws:iam::aws:policy/AdministratorAccess"
```

Comment:
Why Admin access?

Reply:
because I find myself adding so many needed permissions, eventually Raanan said just to give admin, because you never know which services we need it to create S3,SQS,RDS...?